components/brave_wallet/browser/keyring_service.h

```diff
@@ -57,6 +57,8 @@ class KeyringService : public mojom::KeyringService {
   KeyringService(JsonRpcService* json_rpc_service,
                  PrefService* profile_prefs,
                  PrefService* local_state);
+  // Constructor for tests.
+  KeyringService();
```

Comment:
Let's not do this. Just copy-paste deps for instance of KeyringService from some test